base_view_inheritance_extension/tests/test_base_view_inheritance_extension.py

```diff
@@ -0,0 +1,180 @@
+# Copyright 2016 Therp BV <http://therp.nl>
+# Copyright 2021 Camptocamp SA (https://www.camptocamp.com).
+# @author Iván Todorovich <ivan.todorovich@camptocamp.com>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from lxml import etree
+
+from odoo.tests.common import TransactionCase
+
+
+class TestBaseViewInheritanceExtension(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.maxDiff = None
+
+    def test_base_view_inheritance_extension(self):
+        view_id = self.env.ref("base.view_partner_simple_form").id
+        arch, view = self.env["res.partner"]._get_view(view_id=view_id)
```

Comment:
If we don't use it, replace it with **`_`**
```suggestion
        arch, _ = self.env["res.partner"]._get_view(view_id=view_id)
```